src/client.js

```diff
@@ -206,15 +206,32 @@ export default class StitchClient {
    * @param {Object} [args] Arguments to pass to the function.
    */
   executeFunction(name, ...args) {
-    let responseDecoder = (d) => ExtJSON.parse(d, { strict: false });
-    let responseEncoder = (d) => ExtJSON.stringify(d);
-
-    const functionJson = {
+    return this._doFunctionCall({
       name,
       arguments: args
-    };
+    });
+  }
 
-    return this._do('/functions/call', 'POST', { body: responseEncoder(functionJson) })
+  /**
+   * Executes a service.
+   *
+   * @param {String} service The name of the service.
+   * @param {String} action The name of the service action.
+   * @param {Object} [args] Arguments to pass to the service action.
```

Comment:
According to https://stackoverflow.com/questions/4729516/correct-way-to-document-open-ended-argument-functions-in-jsdoc the right way to document the variadic args would look like this:

`@param {...*} args Arguments to pass to the service action.`